README.md

```diff
@@ -21,7 +21,7 @@ The project in is active process of development so new features are coming soon!
 Contribution
 ---------------------------
 We welcome and appreciate any help, even if it's a tiny text or code change. Please read [contribution](https://github.com/ihhub/penguinV/blob/master/CONTRIBUTING.md) page before starting work on a pull request. All contributors are listed in the project's wiki [page](https://github.com/ihhub/penguinV/wiki/Contributors). 
-Not sure what to start with? Feel free to refer to [`good first issue`](https://github.com/ihhub/penguinV/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) or [`help wanted`](https://github.com/ihhub/pen*guinV/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) tags.
```

Comment:
Oh, sorry, my bad! I've been experimenting with styles how it'd be possible to present the links (was thinking to use <kbd>this style</kbd> since it looks somewhat similar to the actual labels), and forgot to double check if the links worked fine.

Reply:
This is fine. I checked all links and they work perfectly :)

Reply:
Are you proposing to change links to use <kbd>blah-blah-blah</kbd> style?

Reply:
I was thinking to make them in this style initially, but changed my mind back to the [hyperlink]() style, just so the README looks consistent. Yet, it won't be a problem to do if you want me to.

Reply:
In my own opinion this style looks pretty good as it highlights that these links are labels. Could you please make such change as I'm currently on phone?

Reply:
Yes, absolutely =)